pages/form.js

```diff
@@ -0,0 +1,16 @@
+import MachineForm from "@/components/MachineForm";
+import { useRouter } from "next/router";
+import { StyledButton } from "@/components/Button/Button.styled";
+
+export default function FormPage() {
+  const router = useRouter();
+
+  return (
+    <>
+      <MachineForm />
+      <StyledButton type="button" onClick={() => router.push("/")}>
```

Comment:
I think for accessibility reasons we would want to use a Link, as this seems to be purely for navigation.

[Further explanation](https://www.digitala11y.com/links-vs-buttons-a-perennial-problem/#:~:text=Use%20buttons%20when%20the%20user,or%20spectators%20of%20the%20site.)